recipes-mender/mender/files/mender.conf

```diff
@@ -1,12 +1,13 @@
 {
-  "pollIntervalSeconds": 60,
-  "ServerURL": "@MENDER_SERVER_URL@",
-  "DeviceID": "DEVICE_ID",
-  "ServerCertificate": "@MENDER_CERT_LOCATION@",
   "ClientProtocol": "https",
+  "DeviceID": "DEVICE_ID",
   "HttpsClient": {
     "Certificate": "",
     "Key": ""
-  }
+  },
+  "PartitionANumber": "@MENDER_PARTITION_A_NUMBER@",
+  "PartitionBNumber": "@MENDER_PARTITION_B_NUMBER@",
```

Comment:
Similar as above, maybe instead of numbers we should use whole strings defining partition name?